platform/ui/src/contextProviders/LabellingFlowProvider.js

```diff
@@ -0,0 +1,131 @@
+import React, {
+  createContext,
+  useContext,
+  useEffect,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import LabellingManager from '../../../viewer/src/components/Labelling/LabellingManager';
```

Comment:
This import is across package boundaries?

Reply:
Actually i think those providers should be located in viewer, right? :boom:

Reply:
Done. Components are now passed through props. thanks.